api/v1/helpers/verbs/utils.js

```diff
@@ -678,6 +678,26 @@ function forbidden(next, modelName) {
   handleError(next, err, modelName);
 } // forbidden
 
+/**
+ * Get IP Address from request.
+ * @param  {Object} req - Request object
+ * @returns {String} - IP Address
+ */
+function getIPAddrFromReq(req) {
+  let ipAddr = null;
+  if (req) {
+    if (req.connection && req.connection.remoteAddress) {
+      ipAddr = req.connection.remoteAddress;
+    }
```

Comment:
Was the intention here for the action taken on line 690 to get overwritten by line 694? i.e. if you get to line 690, should you just return req.connection.remoteAddress, regardless of whether there is an x-forwarded-for header?